src/liff/pages/Source.svelte

```diff
@@ -5,11 +5,12 @@
 
   import { page } from '../lib';
   import { ARTICLE_SOURCE_OPTIONS, SOURCE_PREFIX } from 'src/lib/sharedUtils';
-  import { assertInClient } from '../lib';
+  import { assertInClient, assertSameSearchSession } from '../lib';
 
   let processing = false;
   onMount(() => {
     assertInClient();
+    assertSameSearchSession();
```

Comment:
Should here use `async` `await`?

Reply:
It's not required because there is no action that follows the assertion. However I think it's good to do `await` here just in case we want to add more data loading in the future. I all change it.